bootstrap/node_logic.go

```diff
@@ -63,6 +63,8 @@ func NewNodeLogic(
 	//consensusAlgos = append(consensusAlgos, leanhelix.NewLeanHelixConsensusAlgo(gossipService, blockStorageService, transactionPoolService, consensusContextService, logger, nodeConfig))
 	consensusAlgos = append(consensusAlgos, benchmarkconsensus.NewBenchmarkConsensusAlgo(ctx, gossipService, blockStorageService, consensusContextService, logger, nodeConfig))
 
+	metricFactory.ReportEvery(ctx, nodeConfig.MetricsReportInterval(), logger)
```

Comment:
1) rename variable `metricRegistry`
2) do we really need the report metric interval as a config value?

Reply:
2. I don't see any reason why not. Everything is configurable, and this should be too.